packages/react-magma-dom/src/components/Popover/PopoverTrigger.tsx

```diff
@@ -0,0 +1,150 @@
+import * as React from 'react';
+import { IconButton } from '../IconButton';
+import { Omit, useForkedRef, XOR } from '../../utils';
+import {
+  Button,
+  ButtonProps,
+  ButtonSize,
+  ButtonType,
+  ButtonVariant,
+} from '../Button';
+import { ThemeContext } from '../../theme/ThemeContext';
+import styled from '@emotion/styled';
+import { IconProps, InfoIcon } from 'react-magma-icons';
+import { PopoverContext } from './Popover';
+
+export interface IconOnlyPopoverTriggerProps
+  extends Omit<ButtonProps, 'children'> {
+  /**
+   * Icon to display within the component
+   */
+  icon?: React.ReactElement<IconProps>;
+  /**
+   * The text the screen reader will announce. Required for icon-only buttons
+   */
+  'aria-label': string;
+}
+
+export interface IconTextPopoverTriggerProps extends ButtonProps {
+  /**
+   * Icon to display within the component
+   */
+  icon?: React.ReactElement<IconProps>;
+  /**
+   * The content of the component
+   */
+  children?: React.ReactChild | React.ReactChild[] | string;
+}
+
+const TriggerButtonContainer = styled.div`
+  width: 100%;
+`;
+
+const StyledIcon = styled(IconButton)`
+  border-radius: 8px;
+`;
+
+export type PopoverTriggerProps = XOR<
+  IconOnlyPopoverTriggerProps,
+  IconTextPopoverTriggerProps
+>;
+
+export const PopoverTrigger = React.forwardRef<
+  HTMLButtonElement,
+  PopoverTriggerProps
+>((props, forwardedRef) => {
+  const context = React.useContext(PopoverContext);
+  const theme = React.useContext(ThemeContext);
+
+  const ref = useForkedRef(context.toggleRef, forwardedRef);
+
+  const { icon = <InfoIcon />, children, ...other } = props;
```

Comment:
@orion-cengage do we want to default to the InfoIcon or always expect users to choose?